src/browser/documentation/sidebar-guides/intro.tsx

```diff
@@ -22,31 +22,34 @@ import React from 'react'
 import { BuiltInGuideSidebarSlide } from '../../modules/Carousel/Slide'
 import { isMac } from 'browser/modules/App/keyboardShortcuts'
 
-const title = 'Introduction'
+const title = 'Intro Guide'
 const slides = [
   <BuiltInGuideSidebarSlide key="s1">
+    <h3>Navigating Neo4j Browser</h3>
     <p>
-      Neo4j Browser is a command driven client, like a web-based shell
-      environment. It is perfect for running ad-hoc graph queries, with just
-      enough ability to prototype a Neo4j-based application.
+      Neo4j Browser is a command-driven client as a web-based shell environment.
+      It is perfect for running ad-hoc graph queries, with just enough ability
+      to prototype a Neo4j-based application.
     </p>
     <ul className="big">
+      <li>Developer focused.</li>
+      <li>Used for writing and running Cypher graph queries.</li>
+      <li>Exportable tabular results of any query result.</li>
       <li>
-        Developer focused, for writing and running graph queries with Cypher
-      </li>
-      <li>Exportable tabular results of any query result</li>
-      <li>
-        Graph visualization of query results containing nodes and relationships
+        Graph visualization of query results containing nodes and relationships.
       </li>
+      <li>Convenient exploration of the Neo4j's REST API.</li>
     </ul>
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="s2">
-    <h3>Editor</h3>
-    <p className="lead">Command editing and execution</p>
+    <h3>Browser Editor (a.k.a. Editor)</h3>
+    <p className="lead">
+      <em>Edit and execute Cypher statements and Browser commands</em>
+    </p>
     <p>
-      Editor pane is used to edit and run for Cypher statements and Browser
-      commands. Browser commands begin with <code>:</code>, for example{' '}
-      <code>:help</code>
+      Editor is the primary interface for entering and running Cypher statements
+      and Browser commands. Browser commands begin with <code>:</code>, for
+      example, <code>:help</code>.
```

Comment:
The Editor is the primary interface for entering and running Cypher statements
      and Browser commands. Browser commands begin with <code>:</code>, for
      example, <code>:help</code>.